devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -188,7 +216,14 @@ def parse(String description) {
 		if ((description?.startsWith("catchall:")) || (description?.startsWith("read attr -"))) {
 			def descMap = zigbee.parseDescriptionAsMap(description)
 			if (descMap.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER && descMap.attrInt == 0x0021) {
-				event = getBatteryEvent(zigbee.convertHexToInt(descMap.value))
+				def batteryValue
+				if (isSomfySituo()) {
+					// Somfy Situo reports decimal values
+					batteryValue = descMap.value
```

Comment:
This sounds like a device. Has this been discussed with Somfy?

Reply:
@tpmanley did you mean "device-specific behavior?" or something?

Reply:
Oops, I meant to say "device bug"

Reply:
@greens @tpmanley 
nah, the device reports hex values, i inserted new batteries to verify that (previously i was witnessing only small values from range (0-99).